src/FixedDataTable.js

```diff
@@ -742,7 +742,7 @@ var FixedDataTable = React.createClass({
   _onColumnReorderMove(
     /*number*/ deltaX
   ) {
-    var reorderingData = this.state.columnReorderingData;
+    var reorderingData = JSON.parse(JSON.stringify(this.state.columnReorderingData));
```

Comment:
that's peculiar.  is it just because it changes the reference?  If so can you add a TODO to replace w/ _.cloneDeep?